src/main/java/de/samply/share/broker/rest/Searchbroker.java

```diff
@@ -1271,7 +1275,63 @@ public Response setSiteByName(@HeaderParam(HttpHeaders.AUTHORIZATION) String aut
     }
   }
 
+  /**
+   * Execute structured query.
+   *
+   * @param queryContainer the structuredQuery and the targets
+   * @return response if the query has been created
+   */
+  @POST
+  @Path("/executeSQ")
```

Comment:
I think, I changed my mind. The monitoring endpoint uses `/queries` and `/v2/queries` with create semantics of CRUD. So we should do the same for the structured-query. Maybe `/structured-query/queries`.